server/index.js

```diff
@@ -9,13 +9,17 @@ app.use(express.json());
 
```

Comment:
I like the way you set up your routes, I'll re-evaluate mine for complexity after this